waku/v2/node/wakunode2.nim

```diff
@@ -138,28 +144,43 @@ proc new*(T: type WakuNode, nodeKey: crypto.PrivateKey,
   let
     rng = crypto.newRng()
     hostAddress = tcpEndPoint(bindIp, bindPort)
+    wsHostAddress = tcpEndPoint(bindIp, wsbindPort) & addWsFlag
     announcedAddresses = if extIp.isNone() or extPort.isNone(): @[]
-                         else: @[tcpEndPoint(extIp.get(), extPort.get())]
+                        elif wsFlag == false: @[tcpEndPoint(extIp.get(), extPort.get())]
+                        else : @[tcpEndPoint(extIp.get(), extPort.get()),
+                        tcpEndPoint(extIp.get(), wsBindPort) & addWsFlag]
     peerInfo = PeerInfo.init(nodekey)
     enrIp = if extIp.isSome(): extIp
             else: some(bindIp)
     enrTcpPort = if extPort.isSome(): extPort
                  else: some(bindPort)
     enr = createEnr(nodeKey, enrIp, enrTcpPort, none(Port))
   
-  info "Initializing networking", hostAddress,
-                                  announcedAddresses
-  # XXX: Add this when we create node or start it?
-  peerInfo.addrs.add(hostAddress) # Index 0
+  if wsFlag == true:
+    info "Initializing networking", hostAddress, wsHostAddress,
+                                    announcedAddresses
+    peerInfo.addrs.add(wsHostAddress)
+  else : 
+    info "Initializing networking", hostAddress, announcedAddresses
+    
+  peerInfo.addrs.add(hostAddress)
   for multiaddr in announcedAddresses:
     peerInfo.addrs.add(multiaddr) # Announced addresses in index > 0
   
-  var switch = newStandardSwitch(
-    some(nodekey),
-    hostAddress,
-    transportFlags = {ServerFlags.ReuseAddr},
-    rng = rng,
-    maxConnections = maxConnections)
+  var switch = newWakuSwitch(some(nodekey),
+  hostAddress,
+  wsHostAddress, 
+  transportFlags = {ServerFlags.ReuseAddr},
+  rng = rng, 
+  maxConnections = maxConnections,
+  wsFlag = wsFlag)
+  # TODO Untested - verify behavior after switch interface change
```

Comment:
What is this comment referring to?

Can write `wsFlag` as `wsEnabled` to make intent a bit more clear

Reply:
This is an old comment will remove the same.